applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -98,21 +98,28 @@ func getNumTPUHostsFromTopology(topology string, acceleratorType string) (int32,
 		// v5e TPU VMs can have 1, 4 or 8 chips
 		chipsPerHost, err := strconv.Atoi(acceleratorTypeValues[1])
 		if err != nil {
-			klog.Errorf("Unexpected acceleratorType: %s", acceleratorType)
+			klog.Errorf("RayCluster: %s | Unexpected acceleratorType: %s", clusterName, acceleratorType)
 		}
 		chipsPerHost = min(chipsPerHost, 8) // max of 8 chips per host
 	}
-	return int32(max(chips/chipsPerHost, 1)), nil
+	hosts := int32(max(chips/chipsPerHost, 1))
+	klog.V(1).Infof("RayCluster: %s | GKE topology nodeSelector requests %d TPU VM hosts", clusterName, hosts)
```

Comment:
This might warrant using structured logging. klog supports it using `InfoS` https://pkg.go.dev/k8s.io/klog/v2#InfoS 

```
klog.V(1).InfoS("GKE topology nodeSelector requests TPU VM hosts", "RayCluster", "clusterName", "hosts", hosts)
```

Reply:
Done.